manifest.yml

```diff
@@ -6,12 +6,16 @@ applications:
     buildpacks:
       - nodejs_buildpack
     env:
-      AUTH_BASE: ((AUTH_BASE))
+      AUTH_BASE: https://uat.hsesinfo.org
```

Comment:
I suppose we can change this later (I don't know exactly what the value will be anyway), but this will not be the `AUTH_BASE` for production